snuba/subscriptions/data.py

```diff
@@ -173,27 +197,35 @@ def from_dict(cls, data: Mapping[str, Any]) -> LegacySubscriptionData:
         if not data.get("aggregations"):
             raise InvalidQueryException("No aggregation provided")
 
-        return LegacySubscriptionData(
-            project_id=data["project_id"],
-            conditions=data["conditions"],
-            aggregations=data["aggregations"],
-            time_window=timedelta(seconds=data["time_window"]),
-            resolution=timedelta(seconds=data["resolution"]),
-        )
+        legacy_subs_dict = {
+            "project_id": data["project_id"],
+            "conditions": data["conditions"],
+            "aggregations": data["aggregations"],
+            "time_window": timedelta(seconds=data["time_window"]),
+            "resolution": timedelta(seconds=data["resolution"]),
+        }
+        organization = data.get("organization")
```

Comment:
I don't think you need add `organization` separately. You can add it to the original call.
```
LegacySubscriptionData(
    project_id=data["project_id"],
    conditions=data["conditions"],
    aggregations=data["aggregations"],
    time_window=timedelta(seconds=data["time_window"]),
    resolution=timedelta(seconds=data["resolution"]),
    organization=data.get("organization),
)
```

This is true of all the times you add `organization` in a separate `if`.